R/stats.R

```diff
@@ -167,3 +167,28 @@ run_sea <- function(x, key, years_before = 6, years_after = 4,
 
   return(out_list)
 }
+
+#' A function to calculate the sample depth of an fhx object
+#'
+#' @param x An fhx instance
+#' @return A data.frame containing the years and number of trees
+#'
+#' @export
+#'
+samp_depth <- function(x){
+  require(reshape2)
```

Comment:
We can't do this in packages. All dependencies need to be in the DESCRIPTION file. Luckily, reshape2 is already there!